packages/volto/news/5936.bugfix

```diff
@@ -0,0 +1,2 @@
+Fixed full aligned image size issue on the edit screen by changing padding-left and width
```

Comment:
```suggestion
When adding an image to a block, and selecting the `full` alignment option, keep the image inside the block instead of overflowing. @cihanandac
```

Reply:
@stevepiercy Thank you for the review and suggestion. I have committed the changes for the news.